package.json

```diff
@@ -24,18 +24,24 @@
   "repository": "simonhaenisch/prettier-plugin-organize-imports",
   "homepage": "https://github.com/simonhaenisch/prettier-plugin-organize-imports#readme",
   "peerDependencies": {
-    "@volar/vue-typescript": ">=0.40.2",
+    "@volar/vue-language-plugin-pug": "^1.0.4",
+    "@volar/vue-typescript": "^1.0.4",
```

Comment:
Hm looking at this, maybe it would need to be a major bump after all 🤔

Reply:
Yeah... on the other hand, 3.1.1 is already broken / inkompatibel with volar >= 0.40.2

Reply:
ok true let's just stick with minor (: